ecommerce_api_client/auth.py

```diff
@@ -14,6 +16,8 @@ def __init__(self, username, full_name, email, signing_key, tracking_context=Non
 
     def __call__(self, r):
         data = {
+            'iss': self.issuer,
+            'exp': self.expiration,
```

Comment:
Update this to be `datetime.datetime.utcnow + datetime.timedelta(seconds=self.expires_in)`.